tla-io/src/main/scala/at/forsyte/apalache/io/quint/Quint.scala

```diff
@@ -320,10 +320,26 @@ class Quint(moduleData: QuintOutput) {
         case "replaceAt" => quintArgs => ternaryApp(opName, tla.except)(incrIndexAtNthArg(1, quintArgs))
         case "slice" =>
           quintArgs => ternaryApp(opName, tla.subseq)(incrIndexAtNthArg(1, incrIndexAtNthArg(2, quintArgs)))
+        case "select" =>
+          quintArgs =>
+            binaryApp(opName,
+                (seq, test) => {
+                  val seqType = Quint.typeToTlaType(types(quintArgs(0).id).typ)
+                  val elemType = seqType match {
+                    case SeqT1(elem) => elem
+                    case invalidType => throw new QuintIRParseError(s"sequence ${seq} has invalid type ${invalidType}")
+                  }
+                  val elemParam = tla.param("__e", elemType)
+                  val resultParam = tla.param("__res", seqType)
```

Comment:
In the future, we should try to use a unique name generator

Reply:
There’s a unique name generator on line 49 of this file, which can be dropped in here iiuc.

Reply:
Oh right, thanks for the pointer! I did only see `uniqueLambdaName` until now.

@shonfeder Any reason for not using `tla.pp.UniqueNameGenerator`?